### PR TITLE
Add German translation

### PR DESCRIPTION
**Pull request by Algram**
Added German translation

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
```
Comment (nacho) [inline po/de.po]: mmm this fuzzy should not be here after translating it

File: po/de.po [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2015-03-11 11:24+0100\n"
+"PO-Revision-Date: 2015-03-11 18:29+0100\n"
+"Last-Translator: \n"
```
Comment (nacho) [inline po/de.po]: you are missing here your name and mail

**Comment by Algram** [comment]
Do I have to add my mail?

**Comment by nacho** [comment]
yes, see i.e https://github.com/holylobster/nuntius-linux/blob/master/po/pl.po#L15

**Comment by Algram** [comment]
I updated, as you wished :)

**Comment by prescott66** [comment]
Danke

**Pull request merged**